.github/workflows/build.yml

```diff
@@ -34,12 +36,9 @@ jobs:
         shell: bash
         run: |
           # Update package lists
-          sudo apt update -qq
+          apt update -qq
           # Install build tools
-          sudo apt install -y \
-            cmake \
-            lld \
-            ninja-build
+          apt install -y curl cmake lld ninja-build python3 python3-pip clang
```

Comment:
I think it would look nicer if these went across multiple lines. And again, they should probably be in alphabetical order.

Reply:
but really line by line? that can get so huge

Reply:
Luckily we don't have too many packages to install :D. Also, do we need to explicitly install `curl` and `python3-pip`?

Reply:
ok, and yes we have to